src/git-duet/git-duet-cherry-pick/main.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"git-duet/internal/cmd"
+	"git-duet/internal/cmdrunner"
+)
+
+func main() {
+	err := cmdrunner.Execute(
+		cmd.New("cherry-pick"))
```

Comment:
I may be missing something, but shouldn't this be `cmd.NewWithSignoff("cherry-pick")`?